backend/src/controllers/oficio.controller.ts

```diff
@@ -0,0 +1,21 @@
+import { Controller, Post, Body, Get } from '@nestjs/common';
+import { OficioService } from '../services/oficio.service';
+import { BaseController } from './base.controller';
+import { CreateOficioDTO } from '../dto/oficio.dto';
+
+@Controller('oficios')
```

Comment:
@bianavic 

Vamos manter o nome da entidade "Ofício" como **Legal Letter**.

Poderia renomear as referências?

Reply:
renomeadas